### ISSUE TITLE
Classes are allowed in modules even when classes feature flag is disabled

### ISSUE THREAD

**Issue by nzakas** [eslint/espree#82]
Right now, enabling `ecmaFeatures.modules` means that classes can be default exported:

```js
export default class Foo {}
```

This should throw an error if `ecmaFeatures.classes` is false.

**Comment by nzakas** [comment]
Working on this.
